### ISSUE TITLE
Add (minimal) ProForma support in `mokapot.picked_protein.strip_peptides`

### ISSUE THREAD

**Issue by RalfG** [wfondrie/mokapot#130]
When adding a ProForma sequence with a terminal modification (e.g., `[+304.207].AINEAYK[+304.207]EDYHK[+304.207]`), the `strip_peptide` function does not remove the dash that indicates the N-terminal modification. This breaks peptide-protein mapping downstream (for instance in #114, I think). Can support for ProForma also be added? Thanks!

**Comment by jspaezp** [comment]
Just to be sure ... `[iTRAQ4plex]-EM[Oxidation]EVNES[Phospho]PEK` (which should become `EMEVNESPEK`) is what you mean, right?

**Comment by RalfG** [comment]
Exactly! Using dashes (`-`) instead of periods (`.`) to indicate terminal modifications.
